client/src/index.ts

```diff
@@ -470,3 +481,22 @@ export class XataError extends Error {
 }
 
 export type Links = Record<string, Array<string[]>>;
+
+const isBranchStrategyBuilder = (strategy: BranchStrategy): strategy is BranchStrategyBuilder => {
+  return typeof strategy === 'function';
+};
```

Comment:
Would like input on how you normally use ``typeof`` and/or type guards. I tend to avoid ``typeof`` and only use them inside a type guard.

Reply:
👍 sounds good to me.